docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,128 @@
+# Difference between V2 and V3 Catalog REST APIs
+
+<div class="otp" id="no-index">
+
+### On this page
+- [V3 improvements](#v3-improvements)
+- [Difference between V2 and V3 Catalog APIs](#difference-between-v2-and-v3-catalog-apis)
+- [Related resources](#related-resources)
+
+</div>
+
+V2 and V3 Catalog REST APIs allow you to manage your store's products, categories, and brands, along with their sub-resources.
+
+Both the V2 and V3 Catalog APIs authenticate with OAuth and can be used concurrently within a single application; however, data representation can be significantly different.
+
+V2 and V3 REST APIs are not fully compatible. When resources are available through both APIs, we recommend using the V3 API as it contains performance optimizations and usability improvements.
+For differences in resources, see the [Difference between V2 and V3 Catalog APIs](#difference-between-v2-and-v3-catalog-apis) section of this article.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+> ### Note
+>
+> * BigCommerce will not introduce new resources in V2 APIs.
+> * BigCommerce will eventually migrate all existing V2 resources to V3.
+> * Not all V2 resources have V3 counterparts. 
+
+</div>
+</div>
+</div>
+
+## V3 improvements
+
+* You can perform most tasks with fewer API calls; for example, you can create a product with variants and custom fields in a single request.
+* You can include sub-resources within a request.
+* Each V3 resource includes a meta object, simplifying pagination.
+* V3 [Brands](https://developer.bigcommerce.com/api-reference/store-management/catalog/brands/getbrandbyid), [Categories](https://developer.bigcommerce.com/api-reference/store-management/catalog/category/getcategorybyid), [Products](https://developer.bigcommerce.com/api-reference/store-management/catalog/products/getproductbyid), and [Product Variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-variants/getvariantbyid) expose a metafields resource for use by developers to store custom data.
+* V3 API is optimized for performance. In general, data can be sent, received, and processed faster via V3, relative to V2.
+
+## Difference between V2 and V3 Catalog APIs
+
+### Product variants and modifiers instead of option sets
+
+V3 Catalog introduced a new option model, which included two types of options: [variant options](https://developer.bigcommerce.com/api-docs/store-management/products-overview#variant-options) and [modifiers](https://developer.bigcommerce.com/api-docs/store-management/products-overview#modifier-options). You can now create variants and modifiers in one call without having to create option sets beforehand. The new option model simplified the creation and management of variant prices and modifier adjusters and removed the need to use complex rules, in all but some cases.
+
+#### Variant options
```

Comment:
@ektravel, I'd like to avoid h4 headers, if possible. Is it possible to change these h4s to h3s without convoluting the meaning?

Reply:
@uwikaiddi I've removed h4s and used bold text font instead.